test/tests.scss

```diff
@@ -0,0 +1,4 @@
+// To run Ruby Sass Tests, we need to import all the test spec files here
+@import '_tools.font-size.scss';
+@import '_tools.rem.scss';
+@import '_utilities.widths.scss';
```

Comment:
hmm yea that's a bit weird. do you have this set up locally? maybe try breaking a test case to  see how a failure looks